tools/cellprofiler/test-data/common_image_math.cppipe

```diff
@@ -0,0 +1,94 @@
+CellProfiler Pipeline: http://www.cellprofiler.org
+Version:3
```

Comment:
can we estimate some compatibility things from those versions?

Reply:
not compatible with Version 4 due to the encoding problem :(